src/utils.js

```diff
@@ -332,34 +329,44 @@ export async function getAllMenus() {
   return `*Presto*\n${presto}\n\n*Veglife*\n${veglife}\n\n*Hamka*\n${hamka}\n\n*Click*\n${click}`;
 }
 
+function parsePrestoMenuRow($, row) {
+  return $(row)
+      .find('td')
+      .map((ind, cell) => {
+        if (cell.children[0]) {
```

Comment:
You can replace it with:
`return normalizeWhitespace($(cell).text());`

You can move this function to the top, because it would be used in almost all parsers.